Cargo.toml

```diff
@@ -15,11 +15,14 @@ edition = "2018"
 [dependencies]
 vte_generate_state_changes = { version = "0.1.0", path = "vte_generate_state_changes" }
 arrayvec = { version = "0.5.1", default-features = false, optional = true }
+log = { version = "0.4", optional = true }
 utf8parse = { version = "0.2.0", path = "utf8parse" }
 
 [features]
+ansi = ["log"]
 default = ["no_std"]
-no_std = ["arrayvec"]
+no_alloc = ["arrayvec", "no_std"]
+no_std = []
```

Comment:
What's the reason for changing `no_std` into `no_alloc`? Especially since this is a significant breaking change.

Reply:
I added `no_alloc` feature to use proper structs in no_std environments with allocator. It does not replace `no_std` and should now (in commit 73e79c7) be fully backwards-compatible when `default` feature set is used. If you want, I can invert it to make it opt-in (add `alloc` feature instead)

Reply:
It should be opt-in, but I question its use. I'm not even sure if anyone is interested in running this in `no_std`.

Reply:
If you can say that `osc_raw` in `VtUtf8Receiver` (no_alloc capacity is 1024), `attrs` in `attrs_from_sgr_parameters` (no_alloc capacity is 32) and `buffer` in `SyncState` (no_alloc capacity is 4Kib) very rarely overflow, then `no_alloc` is meaningless. Otherwise, we should provide the possibility to counteract these overflows.